src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      {tabs.length !== 0 && (
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                className={selectedTab.id === tab.id ? 'is-active' : ''}
```

Comment:
Use the classnames library for add classes with condition

Reply:
thanks, fixed!